dist/tools/sliptty/start_network

```diff
@@ -0,0 +1,124 @@
+#! /bin/sh
```

Comment:
probably want to call it `start_network.sh for consistency`

Reply:
Done